website/search/elastic_search.py

```diff
@@ -522,6 +612,17 @@ def update_file(file_, index=None, delete=False):
         )
         return
 
+    if isinstance(target, Preprint):
```

Comment:
A preprint file only shows up in search if it is on a preprint that is verified_publishable. It also must be the primary file. If you change your primary file, the old file will be removed from search.